src/Symfony/Component/Console/Command/HelpCommand.php

```diff
@@ -80,4 +84,17 @@ protected function execute(InputInterface $input, OutputInterface $output)
 
         return 0;
     }
+
+    public function complete(CompletionInput $input, CompletionSuggestions $suggestions): void
+    {
+        if ($input->mustSuggestArgumentValuesFor('command_name')) {
+            $descriptor = new ApplicationDescription($this->getApplication());
+            $suggestions->suggestValues(array_keys($descriptor->getCommands()));
+        }
```

Comment:
Let's add early returns?

```suggestion

            return;
        }
```